bebop_driver/src/bebop_driver_nodelet.cpp

```diff
@@ -90,6 +90,8 @@ void BebopDriverNodelet::onInit()
   const bool param_reset_settings = private_nh.param("reset_settings", false);
   const std::string& param_camera_info_url = private_nh.param<std::string>("camera_info_url", "");
   const std::string& param_bebop_ip = private_nh.param<std::string>("bebop_ip", "192.168.42.1");
+  const int param_bebop_discovery_port = private_nh.param<int>("bebop_discovery_port", 44444);
+  const int param_bebop_d2c_port = private_nh.param<int>("bebop_d2c_port", 43210);  
 
```

Comment:
In addition to using standard int types, I suggest to add a basic safety check here in case the provided port number is not in a valid range (i.e. it's negative).